lib/matplotlib/backends/backend_qt5.py

```diff
@@ -9,8 +14,15 @@
     FigureCanvasBase,  FigureManagerBase, MouseButton, NavigationToolbar2,
     TimerBase, ToolContainerBase, figureoptions, Gcf
 )
+from . import backend_qt as _backend_qt  # noqa
 
 
 @_BackendQT.export
 class _BackendQT5(_BackendQT):
     pass
+
+
+def __getattr__(name):
```

Comment:
This should probably use our convenience stuff in `_api`.

Reply:
I thought about that, but did it "by hand" as we want to avoid the caching that is done by `caching_module_getattr`

Reply:
@QuLogic do you agree this can be resolved?

Reply:
Yes.